server/models/UserModel.py

```diff
@@ -13,13 +13,13 @@ class UserModel(db.Model):
     gmail_address = db.Column(db.String(120), nullable = True)
     gmail_auth_state = db.Column(db.String(120), nullable = True)
     prospects = db.relationship(
-        'ProspectModel', backref='owner', lazy = 'select'
+        'ProspectModel', backref='prospect_owner', lazy = 'select'
     )
     campaigns = db.relationship(
-        'CampaignModel', backref='owner', lazy = 'select'
+        'CampaignModel', backref='campaign_owner', lazy = 'select'
     )
-    email_templates = db.relationship(
-        'EmailTemplateModel', backref='owner', lazy = 'select'
+    templates = db.relationship(
+        'EmailTemplateModel', backref='template_owner', lazy='dynamic'
```

Comment:
If we are using templates instead of email_templates, we might want to choose one of them and be consistent, so far most of my code is using email_template, we might need to discuss this collectively